### PR TITLE
Add logging of environment when cannot find staging folder

### PR DESCRIPTION
**Pull request by TravisEz13**
fall back to a temp folder, needed when running locally

### CODE REVIEW

**Comment by TravisEz13** [comment]
@daxian-dbw

File: vstsBuild/vstsBuild.psm1 [hunk 1]
```diff
@@ -11,6 +11,43 @@ function Clear-VstsTaskState
     $script:taskstate = $succeededStateName
 }
 
+function Get-TempFolder
```
Comment (daxian-dbw) [inline vstsBuild/vstsBuild.psm1:14]: It's a duplicate of a function with the same name in [dockerBasedBuild.common.psm1](https://github.com/PowerShell/PSRelease/blob/693039b63d9f9c2a48de4ca0e6d1535d3b583a24/dockerBasedBuild/dockerBasedBuild.common.psm1#L35). Possible to reuse the same one?

**Review by daxian-dbw — APPROVED** [review]
Leave a comment.

File: vstsBuild/vstsBuild.psm1 (hunk 1 above)
Comment (TravisEz13) [reply vstsBuild/vstsBuild.psm1:14]: I filed an issue. https://github.com/PowerShell/PSRelease/issues/21

**Pull request merged**